src/ui/layout/CreateButton.vue

```diff
@@ -114,7 +75,7 @@ export default {
             return this.openmct.objects.get(this.openmct.router.path[0].identifier)
                 .then((currentObject) => {
                     let legacyContextualParent = this.convertToLegacy(currentObject);
-                    let legacyType = this.openmct.$injector.get('typeService').getType(item.key);
+                    let legacyType = this.openmct.$injector.get('typeService').getType(key);
```

Comment:
we can use the new types api (openmct.types)

Reply:
`CreateAction` (`openmct/platform/commonUI/edit/src/creation/CreateAction.js`) requires `legacyType`.